### PR TITLE
feat: new methods `setDate` / `getDate`

### PR DESCRIPTION
**Pull request by flaviendelangle**
Alternative names: `getDayInMonth` / `setDayInMonth` (can be confusing with `getDaysInMonth`)

### CODE REVIEW

**Comment by dmtrKovalenko** [comment]
Please make sure that CI passes

**Comment by flaviendelangle** [comment]
I feel like Codecov is 1 commit behind
Yesterday I pushed 1 commit with 3-4 lines of code uncovered => no report, just the CI fail
This morning I pushed 1 commit to fix with 0 lines of code uncovered => report saying I have 3-4 lines of code uncovered
Just now I pushed 1 empty commit => report saying I have 0 lines of code uncovered

File: README.md [hunk 1]
```diff
@@ -197,6 +197,9 @@ export interface IUtils<TDate> {
   getSeconds(value: TDate): number;
   setSeconds(value: TDate, count: number): TDate;
 
+  getDate(value: TDate): number;
+  setDate(value: TDate, count: number): TDate;
```
Comment (dmtrKovalenko) [inline README.md:201]: WDYT about making it `getDay`. I think it is more clear in terms of Date is the term for javascript date and may be confusing that it returns a number.

**Review by dmtrKovalenko — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dmtrKovalenko) [reply README.md:201]: Yes getDayInMonths or simply `getDay` would be fine. Everything else is great, so once you will done the rename – we can merge

Comment (flaviendelangle) [reply README.md:201]: I based my naming on the javascript `Date` interface

`new Date().getDay()` => the day in the week
`new Date().getDate()` => the day in the month

But I don't have a strong opinion here
Libraries are not even coherent between them.

Comment (flaviendelangle) [reply README.md:201]: @dmtrKovalenko

**Review by dmtrKovalenko — APPROVED** [review]

**Pull request merged**